### ISSUE TITLE
Mob spawning incorrect in inherited biomes

### ISSUE THREAD

**Issue by KPWhiver** [MCTCP/TerrainControl#275]
Hi,

When I make a new biome that inherits from a default biome, mob spawning behaviour is incorrect. I tested this with forge, without using replacetobiomename. I am not sure whether the issue is also present with bukkit in combination with a forge client.

The version of TC used is build #19. The version of forge is 10.12.1.1060 (Minecraft 1.7.2).

I've made a small testworld with the following modified and inherited files:

WorldConfig.ini:
http://pastebin.com/LtvZActf
DesertMod.bc:
http://pastebin.com/QgVG0suL
DesertMod.bc.inherited:
http://pastebin.com/xZxUadSY

Looking at the inherited file, the issue seems to be that the default mob spawning is used instead of the mob spawning behaviour of the biome that is inherited from.

In short, to reproduce:
- Install forge 10.12.1.1060 
- Install TerrainControl build #19
- Use the provided biomeconfig
- Neutral mobs spawn in the desert, which should not happen

If I've missed some important detail or if you need more information, let me know.

KPW
